module/spl/spl-kmem.c

```diff
@@ -3525,17 +3635,28 @@ kmem_cache_create(
 	/*
 	 * Now that we know the chunk size, determine the optimal slab size.
 	 */
+
+	size_t vquantum = vmp->vm_quantum;
+
+	if ((cflags & KMC_ARENA_SLAB) == KMC_ARENA_SLAB) {
+		VERIFY3U((vmp->vm_cflags & VMC_NO_QCACHE),==,0);
```

Comment:
For posterity, this was wrong, it should be

```
VERIFY3U((vmp->vm_cflags & VMC_NO_QCACHE),==,VMC_NO_QCACHE);
```

and has been corrected in master via force push.